Sources/CryptomatorCloudAccess/Dropbox/DropboxCloudProvider.swift

```diff
@@ -565,10 +571,11 @@ public class DropboxCloudProvider: CloudProvider {
 		if parentCloudPath == CloudPath("/") {
 			return Promise(())
 		}
-		return checkForItemExistence(at: parentCloudPath).then { itemExists -> Void in
+		return checkForItemExistence(at: parentCloudPath).then { itemExists in
 			guard itemExists else {
 				throw CloudProviderError.parentFolderDoesNotExist
 			}
+			return Promise(())
```

Comment:
I don't understand this change. Can it be reverted?

Reply:
Yes, I’ve reverted the change in the same commit 0c1a348.